data/SPRING/projects.yaml

```
@@ -43,8 +43,12 @@ projects:
       - type: Paper
         title: "Synthetic Data -- Anonymisation Groundhog Day"
         url: https://arxiv.org/abs/2011.07018
+        notes:
+          - label: Published at
+            text: "31st USENIX Security Symposium, 2022"
+            url: https://www.usenix.org/conference/usenixsecurity22/summer-accepted-papers
```

I found a more direct link, should we use it?

```suggestion
            url: https://www.usenix.org/conference/usenixsecurity22/presentation/stadler
```

---

Yes, I had that one, but it seems the info is not uniform in the showcase (sometimes it's the conference, sometimes the program, sometimes the actual presentation... :thinking: ). Let's go with your link.

---

I usually would go for the conference or the program, depending on what is available. That's what I mean by it not being uniform. ;-)